pyspark/bigdl/util/common.py

```diff
@@ -179,7 +179,7 @@ class Sample(object):
     def __init__(self, features, label, bigdl_type="float"):
         """
         User should always use Sample.from_ndarray to construct Sample.
-        :param features: a JTensor
+        :param features: a list of JTensors
         :param label: a JTensor
```

Comment:
labels is also a list of JTensors

Reply:
Keep label as one JTensor for now. May need update in the future.